### DISCUSSION TITLE
Using multiple GPUs with Spacy

### DISCUSSION THREAD

**Discussion by cayorodriguez** [explosion/spaCy#8203] [Help: Coding & Implementations] [answered]
Is it possible to use multiple GPUs during training. The --gpu-id variable only accepts single integers, and not a list of ids, like 1,2.

**Comment by polm** [comment, marked as answer]
There is not built-in support for multiple GPUs directly, but it is probably possible with `spacy-ray`. It hasn't been tested much but there's been some recent discussion on it, see https://github.com/explosion/spaCy/issues/8093 for details.
